lib/vdsm/virt/vmdevices/storage.py

```diff
@@ -923,22 +925,54 @@ def get_snapshot_xml(self, snap_info):
 
     def on_block_threshold(self, reported_path):
         """
-        Callback to be executed when we receive a BLOCK_THRESHOLD
-        event from libvirt. We mark this drive accordingly,
-        so the periodic check will pick up this drive for
-        extension.
+        Called when disk allocation excceeds configured threshold.
+
+        Mark this drive for extension, to be picked by the periodic monitor
+        later.
+
+        Return True if threshold state was modified.
         """
         with self._lock:
             if reported_path != self._path:
                 self.log.debug(
                     "block threshold event mismatch drive %r path=%r "
```

Comment:
Start with uppercase (here and lines 945, 949, 966, 971)?

Reply:
This is existing log, not related to this change.